packaging/build_wheels.bash

```diff
@@ -47,14 +47,13 @@ build_wheel_linux() {
     (( $skip )) && return 0
 
     echo " - Installing build requirements"
-    pip install pip auditwheel setuptools
-    pip install -r packaging/build_requirements.txt
+    pip install pip auditwheel
 
     echo " - Building..."
-    rm -rf dist _skbuild
+    rm -rf dist _build
     # Workaround for https://github.com/pypa/manylinux/issues/1309
     git config --global --add safe.directory "*"
-    python setup.py bdist_wheel
+    python -m pip wheel . --wheel-dir dist/ --no-deps
```

Comment:
Might be missing the `-C build-dir=_build` which we deleted a few lines prior.

Reply:
On Linux the issue is not present because the temporary dir is not a symlink (unlike on MacOS, see [this ticket](https://gitlab.kitware.com/cmake/cmake/-/issues/20107)), so it's safe to build it in the temp dir (default if none is specified). If for some reason you need to build a wheel to test locally, you can build it with a persistent dir with just `python -m pip wheel . --wheel-dir dist/ --no-deps -C build-dir=somedir` (the `build_wheels.bash` script is used only in the CI anyway, and will be removed once #1155 is merged).

Reply:
Okay, doesn't answer the question why we're deleting `_build` two lines further up.

Reply:
Guess I forgot to delete that line after copy-pasting from the MacOS one, it doesn't really do anything.